custom/panel_templates/Default/template.php

```diff
@@ -146,12 +146,12 @@ public function onPageLoad() {
 
                     case 'api':
                         $this->addCSSFiles([
-                            (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/core/assets/plugins/toastr/toastr.min.css' => [],
+                            (defined('CONFIG_PATH') ? CONFIG_PATH : '') . '/custom/panel_templates/Default/assets/css/fomantic.min.css' => [],
```

Comment:
Can we not include Formantic in the StaffCP please (even if it is just on a couple of pages)?
There is too much risk of conflicting styles, plus loading two full CSS frameworks is excessive

Reply:
Well, you did not check the file. In there, there are only components and not the whole Fomantic. If you put actual Fomantic it will break

Reply:
> Well, you did not check the file. In there, there are only components and not the whole Fomantic. If you put actual Fomantic it will break

Can we rename the file to something else then? ie formantic.toast.min.css and js to avoid confusion